mkdocs.yml

```diff
@@ -184,3 +185,4 @@ plugins:
             'tools/identity/identitylink.md': 'tools/identitylink/overview.md'
             'tools/identity/self-id.md': 'tools/self-id/overview.md'
     - search
+{"mode":"full","isActive":false}
```

Comment:
Is this expected? Seems strange to have a JSON object in a YAML file.

Reply:
This is not expected. For some reason when I was copying from github's raws. Good catch though!

Reply:
FYI this is still showing up in the current version of the PR for me